src/transformers/models/mobilevit/image_processing_mobilevit.py

```diff
@@ -306,6 +321,48 @@ def preprocess(
         ]
 
         data = {"pixel_values": images}
+
+        if segmentation_maps is not None:
```

Comment:
We should add two new methods to keep in line with other models e.[g. SAM here](https://github.com/huggingface/transformers/blob/976189a6df796a2ff442dd81b022626c840d8c27/src/transformers/models/sam/image_processing_sam.py#L347): 
* preprocess_image
* preprocess_mask

Reply:
Added additional methods to preprocess images and masks seperately in 877b81a4f6a34c24b5156acb19244bb3c8c83663